app/views/devise/registrations/new.html.erb

```diff
@@ -1,29 +1,51 @@
-<h2>Sign up</h2>
+<div class="devise">
+  <h2>Reg&iacute;strate</h2>
+  <%= devise_error_messages! %>
```

Comment:
not sure qué hace esta función o si aquí puede haber un problema, pero se están desplegando dos veces los errores de la form y tal vez puede ser esto lo que lo ocasiona (idk tho), pero ya se despliegan en la línea 8 
`<%= render "devise/shared/error_messages", resource: resource %>`

de hecho creo que devise_error_messages! esta deprecado? [RubyDoc](https://www.rubydoc.info/github/plataformatec/devise/DeviseHelper:devise_error_messages!)

<img width="1251" alt="Screen Shot 2019-03-29 at 10 46 18 PM" src="https://user-images.githubusercontent.com/6772675/55271617-632c2700-5275-11e9-8472-fd6a80ae400d.png">